### PR TITLE
Features/lang vietnamese Add Vietnamese translation

### PR DESCRIPTION
**Pull request by chihung93**
Hello, I spend several time for update sdk to 27 and add vietnamese translation for this library. 
The SliderPageModel class that will resolve some bugs on your library (but too long for remember it ^^).  
Thanks for your distribution to open source.
Have a nice day.

### CODE REVIEW

**Comment by paolorotolo** [comment]
Hi @chihung93, thanks for your pull request.
AppIntro has already been updated to latest SDK, in your commit I only see some repo added in build.gradle (google() was already there) and that is unnecessary.

Could you please open another PR with *just the Vietnamese translation?*
Thanks and have a good day.

File: build.gradle [hunk 1]
```diff
@@ -6,6 +6,7 @@ buildscript {
     repositories {
         google()
         jcenter()
+        google()
```
Comment (paolorotolo) [inline build.gradle]: Google repo was already added

File: build.gradle [hunk 2]
```diff
@@ -19,5 +20,7 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```
Comment (paolorotolo) [inline build.gradle]: Why do we need JitPack here?

File: library/src/main/res/values-vi-rVN/strings.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (paolorotolo) [inline library/src/main/res/values-vi-rVN/strings.xml]: Translations are fine, could you please open another PR with just those?

File: build.gradle (hunk 2 above)
Comment (chihung93) [reply build.gradle]: oops, sorry about this,I have fork it a long time ago and now create PR. I'll create another PR to fix this issue.

File: build.gradle (hunk 1 above)
Comment (chihung93) [reply build.gradle]: Maybe it's conflict when I pulled request and merge with my branch.

**Pull request closed without merging**